project_name/settings.py

```diff
@@ -12,6 +12,8 @@
 
 import os
 
+import dj_database_url
+
```

Comment:
I meant about having two lines between the imports and "code" start. I like separating the import into: standard, third-party and our-own :)